asv_bench/benchmarks/benchmarks.py

```diff
@@ -38,6 +38,13 @@
     trigger_import,
 )
 
+from modin.pandas import Series, Index, DataFrame, MultiIndex
+
+from pandas import (
+    date_range,
+    period_range,
+)
```

Comment:
Don't use these objects here directly. You can access them using `IMPL` var.

This is necessary so that if we want to run the same benchmarks using not Modin, but pandas, we would not need to change the benchmark sources (after all, we can use the times obtained on Modin to compare with pandas).

Reply:
Made changes to use them from IMPL var